packages/pyright-internal/src/tests/harness/fourslash/testState.ts

```diff
@@ -274,6 +274,13 @@ export class TestState {
         return [...this.testData.markerPositions.keys()];
     }
 
+    getStringPositionRange(markerString: string, start = 1, end = 1) {
```

Comment:
I dont think this has anything special about string. either merge this into getPositionRange. or get string node and get string range from it.

ex)
https://github.com/microsoft/pyright/blob/main/packages/pyright-internal/src/analyzer/parseTreeUtils.ts#L1832

Reply:
You're right, I'd still like to keep this though, what about renaming to `getOffsetPositionRange`?

Reply:
any issue with merging this with getPositionRange ?

Reply:
I didn't really want to have to duplicate `getPositionRange('marker', 1, 1)` all over the place but that's not a big deal, happy to merge.